.github/workflows/opt-out-import-test-unit.yml

```diff
@@ -0,0 +1,29 @@
+name: Unit test for opt-out-import
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/opt-out-import-test-unit.yml
+      - optout/**
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+        with:
+          params: |
+            ARTIFACTORY_URL=/artifactory/url
+            ARTIFACTORY_USER=/artifactory/user
+            ARTIFACTORY_PASSWORD=/artifactory/password
+      - name: Run unit tests for opt-out-import lambda
+        working-directory: ./optout
+        run: ../gradlew test
```

Comment:
is it possible to add sonarqube analysis after tests?

Reply:
Sure I'll do that

Reply:
One more question, if don't mind
What happens if jenkins build fails and github action pass? Will I able to merge my changes?

Reply:
For changes to the opt-out-import lambda, there would be no need to run the Jenkins pipeline. The GitHub Actions workflow should run automatically on PR, and the only actual gate to merge is approval from a reviewer.

Reply:
Sonar added but see the error in the run linked below. Is there a sonar task in the gradle build file?

https://github.com/CMSgov/ab2d-lambdas/actions/runs/7658938304/job/20872947370?pr=78#step:6:18

Reply:
So are you able to run sonar tests specifically for opt-out-import lambda in your dev environment? Which command do you run for that, and in which directory?

Reply:
I can't run sonar from `ab2d-lambda/optout`. I assume, working directory should be `ab2d-lambda`.

Locally I've got the same result for `ab2d-lambdas % ./gradlew sonar` and `ab2d-lambdas % ./gradlew sonarrqube`. 
Not sure what's the difference 

I'm fine to hold off sonarqube on the current developing stage